package.json

```diff
@@ -13,6 +13,7 @@
     "chai": ">=1.2.0",
     "cryptojs": ">= 2.5.3",
     "d3": "3.2.7",
+    "eslint": "^1.2.0",
```

Comment:
The caret means that that is essentially `1.*.*`

In our case it doesn't really matter because it's shrinkwrapped so it'll always install 1.2.0 anyways but I don't really see any need to restrict this package when we update the shrinkwrap.